superbench/benchmarks/micro_benchmarks/nvbandwidth.py

```diff
@@ -157,21 +155,29 @@ def _process_raw_line(self, line, parse_status):
         if parse_status['test_name'] and parse_status['benchmark_type'] and matrix_row_pattern.match(line):
             row_data = line.split()
             row_index = row_data[0]
+
             for col_index, value in enumerate(row_data[1:], start=1):
+                # Skip 'N/A' values
+                if value == 'N/A':
+                    continue
+
                 col_header = parse_status['matrix_header'][col_index - 1]
                 test_name = parse_status['test_name']
                 benchmark_type = parse_status['benchmark_type']
                 metric_name = f'{test_name}_cpu{row_index}_gpu{col_header}_{benchmark_type}'
                 parse_status['results'][metric_name] = float(value)
+
             return
 
         # Parse summary results
         summary_match = summary_pattern.search(line)
         if summary_match:
-            value = float(summary_match.group(2))
-            test_name = parse_status['test_name']
-            benchmark_type = parse_status['benchmark_type']
-            parse_status['results'][f'{test_name}_sum_{benchmark_type}'] = value
+            value = summary_match.group(2)
+            # Skip 'N/A' values
```

Comment:
Does 'N/A' means the test case failed?

Reply:
No, just indicate there is no value. For example:
     0       1
0  N/A   1.23
1  1.23   N/A

0->0 or 1->1 is not a valid validation, so there is no value.